tests/test_feature.rb

```diff
@@ -41,7 +41,7 @@ def vdc_lc_state(type=nil)
   # feature test helper
   def feature(feat)
     # Get the feature name string from the yaml
-    ref = cmd_ref.lookup('feature', feat).to_s[/set_value: feature (.*)/]
+    ref = cmd_ref.lookup('feature', feat).to_s[/set_value:.*feature (.*)/]
```

Comment:
`\s+` won't work since this has to match `<state> feature vtp`.  I will change this to optionally match the `<state>` bit.